src/featureMatching/getFastKeypoints.ts

```diff
@@ -29,6 +30,23 @@ export interface GetFastKeypointsOptions extends IsFastKeypointOptions {
    * @default 3
    */
   fastRadius?: number;
+  /**
+   * Algorithm to use to compute corners score.
+   *
+   * @default 'FAST'
+   */
+  scoreAlgorithm?: 'HARRIS' | 'FAST';
+  /**
+   * Options for the Harris score computation.
+   */
+  harrisScoreOptions?: GetHarrisScoreOptions;
+  /**
+   * Should the keypoint scores be normalised between 0 (worst corner) and 1 (best corner).
+   * This feature is only useful if you want to verify the keypoints scores.
+   *
+   * @default false
+   */
+  normaliseScores?: boolean;
```

Comment:
```suggestion
  normalizeScores?: boolean;
```

(replace "normalise" by "normalize" everywhere)